OfficeUIFabric.Demo/src/main/res/layout/activity_avatar_view.xml

```diff
@@ -344,7 +344,8 @@
                 android:layout_gravity="start"
                 app:avatarSize="small"
                 app:avatarStyle="square"
-                app:name="@string/persona_name_erik_nason" />
+                app:name="@string/persona_name_erik_nason"
+                app:backgroundColor="@android:color/holo_blue_bright"/>
```

Comment:
Could you sub this out for one of the "dark" holo colors? The white on bright background isn't looking very accessible.